src/utils/get-linked-prs.ts

```diff
@@ -0,0 +1,44 @@
+import { Context } from "../types/context";
+
+interface GetLinkedParams {
+  owner: string;
+  repository: string;
+  issue?: number;
+  pull?: number;
+}
+
+interface GetLinkedResults {
+  organization: string;
+  repository: string;
+  number: number;
+  href: string;
+}
+
+export async function getLinkedPullRequests(context: Context, { owner, repository, issue }: GetLinkedParams): Promise<GetLinkedResults[]> {
```

Comment:
I hope this works. It worked for me but @gentlementlegen had problems in the past when using it in some tests.

Reply:
I've updated it to include one change made in #5 now checking for `pull_request` in the event. This helps restrict to only PRs and then further checks in the linked pr ensures the PR being closed is from the same repo as the current payload and belongs to the issue assignee

Reply:
The main issue I had was that it was not reliable as `cross-referenced` just meant quoted by another PR that may or may not close it, so ended up evaluating comments that were not relevant. Still baffled that there is not easy way to query linked PRs through the API.
I ended up parsing the body of the linked pull-request itself, but if your approach is more reliable it'd be nice to use it inside the other plugins.

Reply:
Aye my heart was in my mouth the first time I saw it try to close this PR lol. From the QA I performed on test issues it does appear to be better yeah but only with restricting it further to match the payload repo etc